components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/util/I18nEmailUtil.java

```diff
@@ -281,4 +285,44 @@ public static String getNotificationLocale() {
                 ? IdentityUtil.getProperty(I18nMgtConstants.NOTIFICATION_DEFAULT_LOCALE)
                 : I18nMgtConstants.DEFAULT_NOTIFICATION_LOCALE;
     }
+
+    /**
+     * Get the notification template subject, body & footer contents as a byte array.
+     *
+     * @param notificationTemplate  Notification template to get the content
```

Comment:
Shall we add missing fullstops

Reply:
Since this is a statement, not a sentence, its not required to have a full stop.